pkg/process/checks/net.go

```diff
@@ -166,15 +166,28 @@ func (c *ConnectionsCheck) Run(nextGroupID func() int32, _ *RunOptions) (RunResu
 // Cleanup frees any resource held by the ConnectionsCheck before the agent exits
 func (c *ConnectionsCheck) Cleanup() {}
 
-func (c *ConnectionsCheck) getConnections() (*model.Connections, error) {
+func (c *ConnectionsCheck) getConnections(maxConnsPerMessage int) (connections *model.Connections, err error) {
 	tu, err := net.GetRemoteSystemProbeUtil()
 	if err != nil {
 		if c.notInitializedLogLimit.ShouldLog() {
 			log.Warnf("could not initialize system-probe connection: %v (will only log every 10 minutes)", err)
 		}
 		return nil, ErrTracerStillNotInitialized
 	}
-	return tu.GetConnections(c.tracerClientID)
+	for {
+		cnx, more, err := tu.GetConnections(c.tracerClientID, maxConnsPerMessage)
+		if err != nil {
+			return nil, err
+		}
+		if connections == nil {
+			connections = cnx
+		}
+		if !more {
+			return connections, err
+		}
+		connections.Aggregate(cnx)
```

Comment:
Where is this defined? I couldn't find it in agent-payload

Reply:
https://github.com/DataDog/agent-payload/pull/249